fg_lib/include/fg_lib.h

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef __FG_LIB_H__
+#define __FG_LIB_H__
+
+/**
+ * @brief Algorithm for battery state-of-charge estimation ("fuel gauge").
```

Comment:
```suggestion
/**
 * @defgroup fg_lib Battery fuel gauge library
 * @{
 *
 * @brief Algorithm for battery state-of-charge estimation ("fuel gauge").
```
Let's move this up from line 21, for consistency.